spec/views/helpers/custom_links_helper_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
+require 'rails_helper'
+
+RSpec.describe CustomLinksHelper, :type => :helper do
+  describe "custom_links" do
+    it "returns '' when option is empty" do
+      begin
+        CustomSetting['custom_content.header_menus'] = ''
+        expect(helper.custom_links).to eq ''
+      ensure
+        CustomSetting['custom_content.header_menus'] = '[]'
+      end
+    end
+
+    it "returns '' when option is not a valid json" do
+      begin
+        CustomSetting['custom_content.header_menus'] = '["a", "b", "c"]'
+        expect(helper.custom_links).to eq ''
+      ensure
+        CustomSetting['custom_content.header_menus'] = '[]'
+      end
+    end
+
+    it "returns valid html" do
```

Comment:
__[rubocop]__
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

<!-- Comment by [SideCI](https://sideci.com) -->